City-of-Helsinki/drupal-module-helfi-tpr
# UHF-212: Service integration

To test this:

Run unit migration:
- `drush migrate:import tpr_unit`

Run service migration:
- `drush migrate:import tpr_service`

1. Make sure service entities are imported (Content -> TPR - Service).
2. Check that unit services are populated (`SELECT * FROM tpr_unit__services;`)
